### ISSUE TITLE
Feature Request: Adopt NextNote

### ISSUE THREAD

**Issue by sunjam** [nextcloud/notes#183]
Please consider adopting the [NextNote app](https://github.com/brantje/nextnote) from @brantje, formerly [ownNote](https://github.com/Fmstrat/ownnote) by @Fmstrat, and directly integrating it into Nextcloud Notes permanently.  It works amazingly well in Nextcloud 13 and includes a *huge* number of features beyond the current Notes app.  [See this related discussion on the Nextcloud discussion forum for a true Evernote alternative.](https://help.nextcloud.com/t/evernote-replacement/25004)
![screenshot 7](https://user-images.githubusercontent.com/1787238/38115282-4a353712-3360-11e8-8f8d-be5a5ca44649.png)
So close!  Here is a screenshot of this great app in action...
![screenshot 8](https://user-images.githubusercontent.com/1787238/38115288-58196574-3360-11e8-9d99-b71f344e2fae.png)

Directly integrating NextNote could resolve issue #175 #143 #138 #117 #115 #90 #77 #76 #74 #71 #57 
#44 #23 #8

**Comment by muppeth** [comment]
Imo the problem is that notes app is file base while nextnotes is database based. As much as it adds to functionality, at the same time, makes then unencrypted when server side or end2end encryption is used, they cannot be edited by any text editor nor they cannot be synced with nextcloud sync app.

**Comment by nachoparker** [comment]
I am a very happy user of the simple notes app, it does what I want with simplicity and I love that.

I do understand that a more powerful alternative would be very beneficial for many users and for the whole ecosystem... I would like to see this happen

I think it makes more sense to have files backing up our notes than the database. I would suggest to build a WYSIWYG mode into the Notes app in order to have more user friendly edits, but still store everything in markup.

Media can be referenced to files that are stored in Nextcloud, or even URLs, just like on github

Web clipping would be the missing piece on this

**Comment by MichaIng** [comment]
Hmm on the other hand maybe for many users the features of notes app are enough and further implementations would be just overhead for them. Personally I would use additional features of NextNote as well, but Notes does actually enough for me so far.

I don't know about the effort of merging both apps, but I can imagine that it is not too easily done. Perhaps counting all this is an considerable argument against merging, keeping two alternatives with different feature range for users to choose what they really need.

**Comment by ThomasDaheim** [comment]
Somehow I have the feeling a similar discussion already happened some time ago...

I'm working on file support for NextNote (as I did for ownNote) since I also prefer to have my notes in the file system and not the database. However, the main difference for me between the various notes apps is whether they are using markdown or html.

**Comment by jancborchardt** [comment]
See my related comments at https://github.com/nextcloud/notes/issues/193#issuecomment-394509655 and https://github.com/nextcloud/notes/issues/138#issuecomment-386765207

TL;DR Nextcloud Notes is designed for a simple & productive writing experience with no distractions. It’s not [a word processor](https://nextcloud.com/collaboraonline/).

For easier-to-access formatting options, I just opened an issue regarding a context-based formatting popover which appears when marking text: https://github.com/nextcloud/notes/issues/200

**Comment by Fmstrat** [comment]
For reference, I'm back on the Notes bandwagon working on a new cross platform NextNote client: https://github.com/brantje/nextnote/issues/112

While markup would be a great addition to the web interface, a full ground-up WYSIWYG notes client is in the works.

**Comment by jancborchardt** [comment]
@Fmstrat cool – for people who want more WYSIWYG, we will refer them to Nextnote then :) But yeah, ideally integrate with Joplin because they already offer sync with Nextcloud, to not do all the work again.

**Comment by sunjam** [comment]
The problem with Joplin is the notes are [bizarrely formatted once they are
synced into Nextcloud, as mentioned here.](
https://github.com/laurent22/joplin/issues/228)  Also, [Joplin's Jex file
format is not actually documented, but the author says he is open to it](
https://github.com/laurent22/joplin/issues/408).  Perhaps some people in
this thread could send him encouragement!

**Comment by jancborchardt** [comment]
See, that is precisely why with Nextcloud Notes we use plain text files. :) They are futureproof and flexible, and you can use them with basically any editor you want. :rocket:
